media/css/_navigation.scss

```diff
@@ -0,0 +1,127 @@
+.navbar-inner {
+  background: $sidebar-background;
+  border-width: 0 !important;
+}
+
+.navbar-inverse {
+  .navbar-inner {
+    background: $sidebar-background;
+  }
+
+  .brand:hover,
+  .nav > .active > a,
+  .nav > .active > a:hover,
+  .nav > .active > a:focus {
+    background: $sidebar-link-highlight;
+  }
+}
+
+.navbar {
+  .brand {
+    max-height: 50px;
```

Comment:
Properties should be ordered margin-left, margin-right, max-height, overflow, padding